src/SprykerEco/Zed/AmazonQuicksight/Communication/Expander/QuicksightUserFormExpander.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\AmazonQuicksight\Communication\Expander;
+
+use SprykerEco\Zed\AmazonQuicksight\AmazonQuicksightConfig;
+use SprykerEco\Zed\AmazonQuicksight\Dependency\Facade\AmazonQuicksightToTranslatorFacadeInterface;
+use Symfony\Component\Form\DataTransformerInterface;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormEvent;
+use Symfony\Component\Form\FormEvents;
+
+class QuicksightUserFormExpander implements QuicksightUserFormExpanderInterface
+{
+    /**
+     * @var string
+     */
+    protected const FIELD_QUICKSIGHT_USER_ROLE = 'quicksight_user_role';
+
+    /**
+     * @var string
+     */
+    protected const PLACEHOLDER_QUICKSIGHT_USER_ROLE = 'Select user role';
+
+    /**
+     * @var string
+     */
+    protected const FIELD_QUICKSIGHT_USER_ROLE_HELP_MESSAGE = 'Author: A user who can create data sources, datasets, analyses, and dashboards. <br/>Reader: A user who has read-only access to dashboards.';
```

Comment:
```suggestion
    protected const HELP_MESSAGE_QUICKSIGHT_USER_ROLE = 'Author: A user who can create data sources, datasets, analyses, and dashboards. <br/>Reader: A user who has read-only access to dashboards.';
```

Minor: would be more consistent with other constants name